src/index.ts

```diff
@@ -44,8 +46,13 @@ createConnection({
         origin: frontendUrl,
         methods: ['GET', 'POST'],
       },
+      transports: ['websocket', 'polling'],
     });
+    const pubClient = new RedisClient({ url: process.env.REDIS_URL });
+    const subClient = pubClient.duplicate();
 
+    io.adapter(createAdapter({ pubClient, subClient }));
+    io.adapter();
```

Comment:
Wondering why to call adapter second time without parameters? Could you add explaining comment to the code for future developers?

Reply:
Thank you. It's redundant code